version/tailscale-version.sh

```diff
@@ -35,9 +57,7 @@ git clean -d -x -f
 git fetch -q --all --tags
 git checkout -q "$mod_version"
 
-eval $(./build_dist.sh shellvars)
-git_hash=$(git rev-parse HEAD)
-short_hash=$(echo "$git_hash" | cut -c1-9)
-echo ${VERSION_SHORT}-t${short_hash}
```

Comment:
I guess I don't know how this historically tried to work. Ignore me for now but I'll do some digging

Reply:
There's another append in the Makefile for the local commit. I'm working on cleaning that up, but incrementally the step right now is to open up easier diagnostic work without needing to copy/paste patches in and out.